resources/fetchersManager/test_helper.go

```diff
@@ -0,0 +1,131 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
Let's delete this file, see specific comments below.

Reply:
all functions moved to registry_test.go